### DISCUSSION TITLE
How to keep/prevent a message from being deleted/consumed

### DISCUSSION THREAD

**Discussion by talbananaz** [bbc/sqs-consumer#512] [Help]
### Summary

I would like to know how to keep/prevent a message from being deleted/consumed?
I want one machine to just read what is going on (without deleting the message) and another to consume the message.
I have a dead letter queue set up so if I return an empty object or throw it ends up there and that is not the behaviour that I want.

**Comment by nicholasgriffintn** [comment]
There's a setting for this: https://bbc.github.io/sqs-consumer/interfaces/ConsumerOptions.html#shouldDeleteMessages
